src/commands/setup.rs

```diff
@@ -80,13 +81,57 @@ impl Setup {
                     //Configure Bridge and veth_pairs
                     let status_block = network::core::Core::bridge_per_podman_network(
                         per_network_opts,
-                        &network,
+                        network,
```

Comment:
Same there is a type change i guess because of iter.

Reply:
yes